Modules/selectmodule.c

```diff
@@ -2444,12 +2444,19 @@ _select_exec(PyObject *m)
         return -1;
     }
 
+#define ADD_INT(VAL) \
+    do { \
+        if (PyModule_AddIntConstant((m), #VAL, (VAL)) < 0) { \
+            return -1; \
+        } \
+    } while (0)
```

Comment:
Style nit; aligning the line continuation chars are visually less cluttering:

```suggestion
#define ADD_INT(VAL) do {                                 \
    if (PyModule_AddIntConstant((m), #VAL, (VAL)) < 0) {  \
        return -1;                                        \
    }                                                     \
} while (0)
```